api/namex/constants/__init__.py

```diff
@@ -173,6 +176,10 @@ class BCProtectedNameEntityTypes(AbstractEnum):
     PRIVATE_ACT = EntityTypes.PRIVATE_ACT.value
     PARISH = EntityTypes.PARISH.value
     SOCIETY = EntityTypes.SOCIETY.value
+    CONTINUE_IN = EntityTypes.CONTINUE_IN
+    BEN_CONTINUE_IN = EntityTypes.BEN_CONTINUE_IN
+    CCC_CONTINUE_IN = EntityTypes.CCC_CONTINUE_IN
+    ULC_CONTINUE_IN = EntityTypes.ULC_CONTINUE_IN
```

Comment:
Without this, name analysis would fail, ie:

![image](https://github.com/user-attachments/assets/03dfbea7-1822-4479-97b1-80c917b0b86a)